packages/api/orm/entity/Portfolio.ts

```diff
@@ -0,0 +1,42 @@
+import { Application } from "./Application";
+import { CloudServiceProvider } from "../../models/CloudServiceProvider";
+import { Column, Entity, OneToMany } from "typeorm";
+import { ProvisionableEntity } from "./ProvisionableEntity";
+import { TaskOrder } from "./TaskOrder";
+
+@Entity()
+export class Portfolio extends ProvisionableEntity {
+  @Column({ type: String, nullable: false, length: 100 })
+  name: string;
+
+  @Column({ type: String, nullable: true, length: 300 })
+  description: string;
+
+  @Column({ type: String, nullable: false })
+  owner: string;
+
+  @Column({ type: "enum", enum: CloudServiceProvider, nullable: false })
+  csp: CloudServiceProvider;
+
+  @Column({ type: String, nullable: false, array: true })
+  dodComponents: Array<string>;
+
+  @Column({ type: String, nullable: false, array: true })
+  portfolioManagers: Array<string>;
+
```

Comment:
Any reason to prefer `Array<string>` here over `string[]` that gets used later?

Reply:
Nope.  Will make consistent.

Reply:
Our other models tend to use the `Array<type>` syntax, so will use that.

Reply:
Updated in d0cb229be7a30f5bfe8a4e027345dcd518cff62a